### PR TITLE
[Java] chars exercise

### PR DESCRIPTION
**Pull request by ystromm**
https://github.com/exercism/v3/issues/1964

### CODE REVIEW

File: languages/java/exercises/concept/chars/.docs/instructions.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+### 2. Replace control characters with the upper case string "CTRL"
+
+Modify the (_static_) `Identifier.clean()` method to replace control characters with the upper case string `"CTRL"`.
+
+```java
+Identifier.clean("my\0Id");
+// => "myCTRLId",
+```
+
+### 3. Convert kebab-case to camelCase
```
Comment (mirkoperillo) [inline languages/java/exercises/concept/chars/.docs/instructions.md:27]: Section 3 and 4 is still in C# style

File: languages/java/exercises/concept/chars/src/test/java/IdentifierTest.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class IdentifierTest {
+
+    @Test
+    public void empty() {
+        assertThat(Identifier.clean("")).isEmpty();
+    }
+
+    @Ignore("")
```
Comment (mirkoperillo) [inline languages/java/exercises/concept/chars/src/test/java/IdentifierTest.java:13]: This advice is for all `@Ignore` annotations in the file.
To be aligned to Java track style, we use this `@Ignore("Remove to run test")`

**Comment by ystromm** [comment]
Thanks for the comments. Think I understood everything. 

I noticed the chars exercise was missing the design.md and config.json files so I added them. Not sure if they are mandatory.

File: languages/java/exercises/concept/chars/.docs/instructions.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+In this exercise you will implement a partial set of utility routines to help a developer
+clean up identifier names.
+
+In the 4 tasks you will gradually build up the routine `Clean` A valid identifier comprises
```
Comment (mirkoperillo) [inline languages/java/exercises/concept/chars/.docs/instructions.md:4]:
```suggestion
 In the 4 tasks you will gradually build up the routine `clean` A valid identifier comprises 
```

**Comment by mirkoperillo** [comment]
LGTM.
I suggest only the fix of a typo and the PR is ready to be merged.
Great work and thank you for the help

**Comment by ystromm** [comment]
Thanks! The instructions are updated and I added an additional test for good measure. I don't think I have permission to merge (just a mentor).

**Comment by mirkoperillo** [comment]
good @ystromm I merge it now

**Pull request merged**